sdk/python/kfp/utils.py

```diff
@@ -0,0 +1,61 @@
+def use_secret(secret_name:str=None, volume_name:str=None, secret_volume_mount_path:str=None, env_variable:str=None, secret_file_path_in_volume:str=None):
+    """    
+       An operator that configures the container to use a secret.
+       
+       This assumes that the secret is created and availabel in the k8s cluster.
+    
+    Keyword Arguments:
+        secret_name {String} -- [Required] The k8s secret name (default: {None})
+        volume_name {String} -- The pod volume name (default: {None})
+        secret_volume_mount_path {String} -- [Required] The path to the secret that is mounted  (default: {None})
+        env_variable {String} -- Env variable pointing to the mounted secret file. Requires both the env_variable and secret_file_path_in_volume to be defined. 
+                                 The value is the path to the secret (default: {None})
+        secret_file_path_in_volume {String} -- The path to the secret in the volume. This will be the value of env_variable. 
+                                 Both env_variable and secret_file_path_in_volume needs to be set if any env variable should be created (default: {None})
+    
+    Raises:
+        ValueError: If not the necessary variables (secret_name, volume_name", secret_volume_mount_path) are supplied.
+                    Or only one of  env_variable and secret_file_path_in_volume are supplied
+    
+    Returns:
+        [ContainerOperator] -- Returns the container operator after it has been modified. 
+    """
+
+    if secret_name:
```

Comment:
If the user pass in an empty secret_name, shall we error out or just put `'_volume'` as the volume name?

Reply:
I think we should error out.

Reply:
hmm changed my mind I would like to leave it and don't have any opinon on what name the user sets the "secret_name" to.

Reply:
Okay sounds good.  My two cents here: if `secret_name == None or ''` this assignment to volume_name won't happen. Do we have any treatment to this condition?

Reply:
True, I will updated with the else statement to set the volume_name to "_volume"

Reply:
I think the secret name must not be empty. Kubernetes does not support secrets with empty names.

Reply:
I will change it and update so i assert for name and raise a ValueError if there name is a empty string

Reply:
Is it the same for "secret_volume_mount_path"?